src/children.js

```diff
@@ -3,21 +3,43 @@
  * @param {object | object[]} node or nodes
  * @returns {object[]}
  */
-export default function children (node) {
+export function of (node) {
```

Comment:
Btw what is the rationale to not have a `children()` function anymore and turn this into just a namespace?

Reply:
Rationale was so that we could have function that returns just a list of children, and one that returns a list of the metadata objects. However, if you think there's no longer reason to have the one that just returns children, we can get rid of that old one and switch it back to a default export

Reply:
I do think it's nice to have the option to use either (either the complete children function with the additional info, or the simple one that just returns a list of nodes)

Reply:
I don't see how having a default export precludes this. The default export would continue to return nodes only, and `children.paths()` can return the paths.

Reply:
We could even make the default export *also* a named export, to facilitate `import * as `children`:

```js
export default function children() { ... }
children.of = children;
export { of: children.of }
```

Reply:
> We could even make the default export _also_ a named export, to facilitate `import * as children`:
> 
> ```js
> export default function children() { ... }
> children.of = children;
> export { of: children.of }
> ```

I'm happy with this!

Reply:
Could you explain what the difference would be between what you wrote above and 
```js
export default function children() { ... }
export {of: children}
```?

Reply:
I think what you proposed is also fine. Better actually, since to do what I had in mind, you'd need to attach *all* the named exports on the default exported function, and then we do lose out on tree shaking.